src/scss/_color-use-cases.scss

```diff
@@ -4,4 +4,5 @@
 @include oColorsSetUseCase(o-table-row-alt, background, 'warm-1');
 @include oColorsSetUseCase(o-table-header-row, border, 'pink-tint4');
 @include oColorsSetUseCase(o-table-header, color, 'cold-3');
-@include oColorsSetUseCase(o-table-data, color, 'grey-tint4');
\ No newline at end of file
+@include oColorsSetUseCase(o-table-data, color, 'grey-tint4');
```

Comment:
There's a warm or cold alternative to grey-tint4 I think, if so, better to use that one